js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorerSidebar.tsx

```diff
@@ -161,8 +265,33 @@ export const AssetGraphExplorerSidebar = React.memo(
     }, [indexOfLastSelectedNode, rowVirtualizer]);
 
     return (
-      <div style={{display: 'grid', gridTemplateRows: 'auto minmax(0, 1fr)', height: '100%'}}>
-        <Box flex={{direction: 'column'}} padding={8}>
+      <div style={{display: 'grid', gridTemplateRows: 'auto auto minmax(0, 1fr)', height: '100%'}}>
```

Comment:
It sorta feels like this grid is behaving just like a flexbox where the last item has `flex:1`?

Reply:
I tried that but it had weird issues for some reason, did a little investigation but couldn't crack it.